pkg/apis/fleet/v1alpha1/types.go

```diff
@@ -76,6 +76,8 @@ type PluginConfig struct {
 	// Grafana defines the configuration for the grafana installation and observation.
 	// +optional
 	Grafana *GrafanaConfig `json:"grafana,omitempty"`
+	// Kyverno defines the configuration for the kyverno installation.
+	Kyverno *KyvernoConfig `json:"kyverno,omitempty"`
```

Comment:
nit:  `PolicyConfig`, we should prevent exposing concrete plugin within the Fleet API, otherwise it can be very tricky if we want to replace it in future